src/Utils/MockData/BackendMock.js

```diff
@@ -1,11 +1,19 @@
-import Hierarchy from "./TransportForLondon/Hierarchy.js";
+/*import Hierarchy from "./TransportForLondon/Hierarchy.js";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3ACommentedCode)